charts/rancher-gatekeeper-operator/v3.1.0-beta.700/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+description: A Helm chart for Gatekeeper
+name: gatekeeper-operator
+keywords:
+  - open policy agent
+version: v3.1.0-beta.7
```

Comment:
We should consider adding `00` to the end of this in case we need to make changes to the chart before the app version changes.

Reply:
Depending on how this is implemented in the UI, we might need to add a [questions.yaml](https://github.com/rancher/system-charts/blob/dev/charts/rancher-istio/1.4.400/questions.yaml#L2) to label this correctly. Apparently doesn't work with monitoring because of how it's implemented in the UI, so I'm not sure about this case.